src/index.html

```diff
@@ -17,6 +31,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logotype"
+          class="logo"
+        />
+      </a>
+      <nav class="nav_list">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
```

Comment:
The `data-qa` attribute is used for testing purposes. Ensure it is necessary for your use case or remove it if not needed.